envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/listeners/filters/RBACFilterFactoryTest.kt

```diff
@@ -221,7 +221,7 @@ internal class RBACFilterFactoryTest {
         // given
         val policyName = "IncomingEndpoint(path=/example, pathMatchingType=PATH, methods=[GET, POST], " +
             "clients=[ClientWithSelector(name=client1, selector=null), ClientWithSelector(name=client2, selector=null)], " +
-            "unlistedClientsPolicy=LOG)"
+            "unlistedClientsPolicy=LOG, oauth=null)"
```

Comment:
Maybe we do not need null fields in policy names? WDYT?

Reply:
Perhaps this become more useful when actions in RBAC filter will be added in following PR